ada/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+
+class RedirectUrl(models.Model):
+
+    url = models.URLField(
+        max_length=255,
+        blank=True,
+        null=True,
+        default="https://hub.momconnect.za/confirmredirect",
+    )
+    content = models.TextField(default="This entry has no copy")
```

Comment:
We don't seem to be using this `content` field anywhere except for in the `__str__` method. What is it for?

Reply:
Content returns the copy of the content for that URL. I can't attach an image with this comment so I'll share in Slack. It defaults to "...entry has no copy"

Reply:
I see, the content is the content of the message that contained this URL, it's for us to easily see that information in the django admin